src/Swarm/TUI/Controller.hs

```diff
@@ -161,6 +162,14 @@ handleNewGameMenuEvent scenarioStack@(curMenu :| rest) s = \case
     continue $ s & uiState . uiMenu .~ NewGameMenu (menu' :| rest)
   _ -> continueWithoutRedraw s
 
+-- | Reset the UI state when beginning a new game.
+resetUIState :: UIState -> UIState
+resetUIState =
+  (uiFocusRing .~ initFocusRing)
+    . (uiInventory .~ Nothing)
+    . (uiShowFPS .~ False)
+    . (uiShowZero .~ True)
```

Comment:
Perhaps we could reset the tps to the default value, e.g. `lgTicksPerSecond .~ initLgTicksPerSecond`

Reply:
Oh, good catch!